Improve API docs (#159)
ballerina-platform/module-ballerina-mime: mime-ballerina/Package.md

## Package Overview

> The MIME library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.

Reviewer: ```suggestion
The `mime` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
```

> It provides a set of APIs to work with messages which follow the Multipurpose Internet Mail Extensions

Reviewer: ```suggestion
It provides a set of APIs to work with messages, which follow the Multipurpose Internet Mail Extensions
```

(MIME) specification as specified in the [RFC 2045 standard](https://www.ietf.org/rfc/rfc2045.txt)

## Report Issues

> To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina 

Reviewer: ```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina 
```

> Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).

Reviewer: ```suggestion
Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

## Package Overview

The MIME library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.

> It encapsulates multiple body parts such as attachments into a single message. The communication of such messages 

Reviewer 2: Shall we add a bit more context before this. "It encapsulates multiple body parts" looks like the middle of a description.

follow the MIME (Multipurpose Internet Mail Extensions) specification as specified in the 
[RFC 2045 standard](https://www.ietf.org/rfc/rfc2045.txt)

For more information go to [The MIME Package](https://ballerina.io/learn/api-docs/ballerina/mime/).

> For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).

Reviewer 3: Remove the link.
